gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -0,0 +1,728 @@
+"""Script to create release sites HT for exomes."""
+import argparse
+import json
+import logging
+from copy import deepcopy
+from datetime import datetime
+from functools import reduce
+from typing import Dict, List, Union
+
+import hail as hl
+from gnomad.resources.grch38.reference_data import (
+    dbsnp,
+    lcr_intervals,
+    seg_dup_intervals,
+)
+from gnomad.utils.annotations import region_flag_expr
+from gnomad.utils.slack import slack_notifications
+from gnomad.utils.vcf import AS_FIELDS, SITE_FIELDS
+from hail.typecheck import anytype, nullable, sequenceof
+
+from gnomad_qc.slack_creds import slack_token
+from gnomad_qc.v4.annotations.insilico_predictors import get_sift_polyphen_from_vep
+from gnomad_qc.v4.create_release.create_release_utils import (
+    DBSNP_VERSION,
+    GENCODE_VERSION,
+    MANE_SELECT_VERSION,
+    POLYPHEN_VERSION,
+    SEQREPO_VERSION,
+    SIFT_VERSION,
+    VRS_PYTHON_VERSION,
+    VRS_SCHEMA_VERSION,
+    remove_missing_vep_fields,
+)
+from gnomad_qc.v4.resources.annotations import (
+    get_freq,
+    get_info,
+    get_insilico_predictors,
+    get_vep,
+    get_vrs,
+)
+from gnomad_qc.v4.resources.basics import calling_intervals, qc_temp_prefix
+from gnomad_qc.v4.resources.constants import CURRENT_RELEASE
+from gnomad_qc.v4.resources.release import included_datasets_json_path, release_sites
+from gnomad_qc.v4.resources.sample_qc import interval_qc_pass
+from gnomad_qc.v4.resources.variant_qc import final_filter
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("create_release_ht")
+logger.setLevel(logging.INFO)
+
+# Remove InbreedingCoeff from allele-specific fields because it is processed separately
+# from the other fields.
+AS_FIELDS = deepcopy(AS_FIELDS)
+AS_FIELDS.remove("InbreedingCoeff")
+SITE_FIELDS = deepcopy(SITE_FIELDS)
+TABLES_FOR_RELEASE = [
+    "dbsnp",
+    "filters",
+    "freq",
+    "info",
+    "region_flags",
+    "in_silico",
+    "vep",
+    # "joint_faf",
+]
+
+INSILICO_PREDICTORS = ["cadd", "revel", "spliceai", "pangolin", "phylop"]
+
+FINALIZED_SCHEMA = {
+    "globals": [
+        "freq_meta",
+        "freq_index_dict",
+        "freq_meta_sample_count",
+        "faf_meta",
+        "faf_index_dict",
+        # "joint_freq_meta",
+        # "joint_freq_index_dict",
+        # "joint_freq_meta_sample_count",
+        # "joint_faf_meta",
+        # "joint_faf_index_dict",
+        "age_distribution",
+        "downsamplings",
+        "filtering_model",
+        "inbreeding_coeff_cutoff",
+        "tool_versions",
+        "vrs_versions",
+        "vep_globals",
+        "date",
+        "version",
+    ],
+    "rows": [
+        "freq",
+        "grpmax",
+        "faf",
+        "gen_anc_faf_max",
```

Comment:
I was confused about the difference between the `faf95` field within the `grpmax` struct and the `faf95_max` field in this `gen_anc_faf_max` struct. From chatting with Kristen, I learned that `grpmax.faf95` is the FAF of the gen anc group with the highest **AF** for a variant, and the `gen_anc_faf_max.faf95_max` is the FAF of the gen anc group with the highest **FAF** for a variant. These values are not necessarily the same; the group with the highest AF for a variant isn't necessarily the group with the highest FAF for the variant.

I then asked Anne if there is any utility for the `grpmax.faf95`, and she said no -- I vote we remove `grpmax.faf95` from our release HTs

Reply:
I agree, it was very confusing when I was writing the `gen_anc_faf_max` function.

Reply:
Should we rename it to fafmax? I could just do this in this release code for now

Reply:
I like this, particularly since this struct contains fields noting which gen anc group the FAF95/FAF99 values are from

Reply:
Just to be sure, this == `gen_anc_faf_max` ?

Reply:
yes: I like your proposal of renaming `gen_anc_faf_max` to simply `fafmax`, particularly since `gen_anc_faf_max` already contains fields like `faf95_max_gen_anc` which indicate the genetic ancestry group associated with the fafmax values.

tl;dr -- unless anyone has strong opinions, let's rename `gen_anc_faf_max` to `fafmax` and drop `faf95` from `grpmax`